src/components/IndividualArticle.jsx

```diff
@@ -106,10 +108,15 @@ const IndividualArticle = () => {
             <p className={styles.p__commentCount}>{article.comment_count}</p>
           </div>
 
-          <div className={styles.link__topic}>
-            <TfiTag className={styles.svg__topicTag} />
-            <h4 className={styles.h4__articleTopic}>{article.topic}</h4>
-          </div>
+          <Link
```

Comment:
Like this, although i think theres some duplicattion of the votes component adn this.